cmd/skywire-cli/commands/visor/start.go

```diff
@@ -0,0 +1,45 @@
+package visor
+
+import (
+	"os/user"
+
+	"github.com/bitfield/script"
+	"github.com/spf13/cobra"
+)
+
+var sourcerun bool
+
+func init() {
+	usrLvl, err := user.Current()
+	if err != nil {
+		panic(err)
+	}
+	var root bool
+	if usrLvl.Username == "root" {
+		root = true
+	}
+	//enforce root permissions for VPN operability
+	if root {
+		RootCmd.AddCommand(
+			startCmd,
+		)
+		startCmd.Flags().BoolVarP(&sourcerun, "src", "s", false, "'go run' external commands from the skywire sources")
+	}
+}
+
+var startCmd = &cobra.Command{
```

Comment:
Does not run for me. Just prints out the help if I run `skywire-cli visor start`.

Reply:
the subcommand was only available as root at the time you tested it. I've rewritten to support starting skywire at the user level.